app/presenters/race_edition_presenter.rb

```diff
@@ -16,10 +15,33 @@ def category_size_map
     categories.map { |category| [category.name, grouped_race_entries[category.name]&.size || 0] }
   end
 
+  def edition_count
+    return nil unless year.present?
+
+    year - 2004
+  end
+
+  def short_description
+    case race&.name
+    when "Rattlesnake Ramble Trail Race"
+      "Fowler Trail First"
+    when "Rattlesnake Ramble Trail Race Even-Year"
+      "Eldorado Trail First"
+    when "Rattlesnake Ramble Kids Race"
+      "Kids Race"
+    else
+      nil
+    end
+  end
+
   def sorted_race_entries
     race_entries.joins(:racer).order(sort_param).map { |re| RaceEntryPresenter.new(re) }
   end
 
+  def year
+    date&.year
```

Comment:
What's the & operator do?

Reply:
That's a safe-navigation operator. If `date` does is `nil` for whatever reason, it just returns `nil` instead of raising a `NoMethodError`.